docs/ops.md

```diff
@@ -0,0 +1,33 @@
+# Operational Docs
+
+## Cleaning up a failed migration
+
+Use these instructions if a migration fails, and you want to retry it
+(e.g. you found and fixed the issue that caused it to fail).
+*note: if the migrator called `update` on the migration instance, this could get much more complicated, and fixing forward might be better*
```

Comment:
What do you mean by fixing forward?  Is this referring to steps 3 and later, or something else?

Reply:
fixing forward as in finish the migration by hand